src/realm/geospatial.hpp

```diff
@@ -34,16 +36,14 @@ class Obj;
 class TableRef;
 
 struct GeoPoint {
-    GeoPoint()
-        : longitude(0)
-        , latitude(0)
-        , altitude(c_null_altitude)
+    GeoPoint() // FIXME try to remove?
```

Comment:
It is possible to remove this, I have done it in a local commit, but let's try to merge this PR first and I will deal with the conflicts.